test/__snapshots__/index.spec.js.snap

```diff
@@ -335,22 +344,6 @@ Array [
     "pages/en/getStaticProps/static.json",
     "_next/data/build-id/en/getStaticProps/static.json",
   ],
-  Array [
```

Comment:
Introduced in a5b80475a89f5ab2266059ad2e96c8786ff41421 (app dir)

Reply:
OK, this is because we no longer move ISR pages, because they're needed for the first request. https://github.com/netlify/next-runtime/blob/main/packages/runtime/src/helpers/files.ts#L158